go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/Masterminds/semver/v3 v3.1.1
 	github.com/ProtonMail/go-crypto v0.0.0-20210428141323-04723f9f07d7
 	github.com/cyphar/filepath-securejoin v0.2.2
+	github.com/elazarl/goproxy v0.0.0-20180725130230-947c36da3153
```

Comment:
This looks old. Looks like goproxy hasn't release a new tagged version since 2018.
We should use the latest code in the `master` branch.
Can you edit this line manually to:

```go
github.com/elazarl/goproxy master
```

Save it, even if your editor complains about it and run `go mod tidy -v`.
This would result in go mod fetching the latest revision from `master` branch and updating the above line to be something like:

```go
github.com/elazarl/goproxy v0.0.0-20211114080932-d06c3be7c11b
```